overlays/custom-packages/waypipe/default.nix

```diff
@@ -1,17 +0,0 @@
-# Copyright 2022-2024 TII (SSRC) and the Ghaf contributors
-# SPDX-License-Identifier: Apache-2.0
-{
-  final,
-  prev,
-}:
-# Waypipe with vsock and window borders
-prev.waypipe.overrideAttrs (_prevAttrs: {
-  src = final.pkgs.fetchFromGitLab {
-    domain = "gitlab.freedesktop.org";
-    owner = "mstoeckl";
-    repo = "waypipe";
-    rev = "ca4809435e781dfc6bd3006fde605860c8dcf179";
-    sha256 = "sha256-tSLPlf7fVq8vwbr7fHotqM/sBSXYMDM1V5yth5bhi38=";
-  };
-  patches = [./waypipe-window-borders.patch];
```

Comment:
do we not still need the window boarder coloring?

Reply:
You are right. I missed that.